homeassistant/const.py

```diff
@@ -659,6 +659,9 @@ class Platform(StrEnum):
 DATA_RATE_MEBIBYTES_PER_SECOND: Final = "MiB/s"
 DATA_RATE_GIBIBYTES_PER_SECOND: Final = "GiB/s"
 
+# Fuel consumption units
+FUEL_CONSUMPTION_LITERS_PER_HUNDRED_KILOMETERS = "L/100km"
```

Comment:
Let's wait with adding new common constants. Please move this back. We can see if we should add this later separately.